### PR TITLE
Guest login removed and password handling improved

### PR DESCRIPTION
**Pull request by Adictya**

### CODE REVIEW

**Comment by searchformyusername** [comment]
@L04DB4L4NC3R @Geek-ubaid

File: src/components/loginPane.js [hunk 1]
```diff
@@ -44,19 +43,25 @@ class loginPane extends Component {
 	ChangeHandler(e) {
 		e.target.className = formStyle.InputField;
 	}
-	handleGuest = () =>{
-		localStorage.setItem('Guest', "Yes");
-		localStorage.setItem("theme",JSON.stringify({mode:'light'}));
-	}
+	handleGuest = () => {
```
Comment (L04DB4L4NC3R) [inline src/components/loginPane.js]: If guest login is removed then why is this being done?

File: src/context/MyProvider.js [hunk 2]
```diff
@@ -70,109 +71,125 @@ class MyProvider extends Component {
...
+		return new Promise((resolve, reject) => {
+			fetch(this.apiUrl + "preview/createFile/", {
+				method: "PATCH",
+				headers: {
+					Authorization: this.token,
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({
+					fileName: "New Document",
+					userId: this.userId,
```
Comment (L04DB4L4NC3R) [inline src/context/MyProvider.js]: Why is the userId being sent in the request payload?

File: src/context/MyProvider.js [hunk 3]
```diff
@@ -70,109 +71,125 @@ class MyProvider extends Component {
...
+			fetch(this.apiUrl + "preview/createFile/", {
+				method: "PATCH",
+				headers: {
+					Authorization: this.token,
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({
 					fileName: title,
-					time: "Today",
+					userId: this.userId,
```
Comment (L04DB4L4NC3R) [inline src/context/MyProvider.js]: Here also

Comment (L04DB4L4NC3R) [reply src/context/MyProvider.js]: Token is being used in the backend to get the userId, no point in sending this here

**Review by L04DB4L4NC3R — CHANGES_REQUESTED** [review]

File: src/context/MyProvider.js (hunk 3 above)
Comment (Adictya) [reply src/context/MyProvider.js]: Removed

File: src/components/loginPane.js (hunk 1 above)
Comment (Adictya) [reply src/components/loginPane.js]: Commented

File: src/context/MyProvider.js (hunk 2 above)
Comment (Adictya) [reply src/context/MyProvider.js]: Removed

**Pull request merged**